conformance/results/contour.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: gateway.networking.k8s.io/v1alpha1
+date: "2023-07-25T22:11:56Z"
+gatewayAPIVersion: TODO
```

Comment:
We have #2197 to automate adding the `gatewayAPIVersion`, for now just manually enter it

(I assume it's `v0.7.1`?)

```suggestion
gatewayAPIVersion: v0.7.1
```

Reply:
+1

Reply:
I used a commit off of main for the tests and CRDs+webhook manifest